Plots/Panels/NCL_panel_7.py

```diff
@@ -0,0 +1,162 @@
+U = ds.U
+
+###############################################################################
+# Plot:
+
+# Make three panels (i.e. subplots in matplotlib)
+# Specify ``constrained_layout=True`` to automatically layout panels, colorbars and axes decorations nicely.
+# See https://matplotlib.org/tutorials/intermediate/constrainedlayout_guide.html
+# Generate figure and axes using Cartopy projection
+projection = ccrs.PlateCarree()
+fig, ax = plt.subplots(3,
+                       1,
+                       constrained_layout=True,
+                       subplot_kw={"projection": projection})
+
+# Set figure size (width, height) in inches
+fig.set_size_inches((6, 9.6))
+
+# Add continents
+continents = cartopy.feature.NaturalEarthFeature(name="coastline",
+                                                 category="physical",
+                                                 scale="50m",
+                                                 edgecolor="None",
+                                                 facecolor="lightgray")
+[axes.add_feature(continents) for axes in ax.flat]
+
+# Specify which contour levels to draw
+levels = np.arange(-10, 40, 5)
+
+# Specify locations of the labels
+labels = np.arange(0, 40, 10)
+
+# Using a dictionary makes it easy to reuse the same keyword arguments twice for the contours
+kwargs = dict(
+    levels=levels,  # contour levels specified outside this function
+    xticks=np.arange(-180, 181, 30),  # nice x ticks
+    yticks=np.arange(-90, 91, 30),  # nice y ticks
+    transform=projection,  # ds projection
+    add_colorbar=False,  # don't add individual colorbars for each plot call
+    add_labels=False,  # turn off xarray's automatic Lat, lon labels
+    colors="black",  # note plurals in this and following kwargs
+    linestyles="-",
+    linewidths=0.5,
+)
+
+# Set contour labels, titles, text box and ticks for all panels
+for axes in ax.flat:
+    # Contour-plot U data (for borderlines)
+    contour = U.plot.contour(
+        x="lon",
+        y="lat",
+        ax=axes,
+        **kwargs,
+    )
+    # Label the contours and set axes title
+    axes.clabel(contour, labels, fontsize="small", fmt="%.0f")
+
+    # Add lower text box
+    axes.text(0.995,
+              0.03,
+              "CONTOUR FROM -10 TO 35 BY 5",
+              horizontalalignment='right',
+              transform=axes.transAxes,
+              fontsize=8,
+              bbox=dict(boxstyle='square, pad=0.25',
+                        facecolor='white',
+                        edgecolor='gray'),
+              zorder=5)
+
+    # Use geocat.viz.util convenience function to add left and right title to the plot axes.
+    gvutil.set_titles_and_labels(axes,
+                                 lefttitle="Zonal Wind",
+                                 lefttitlefontsize=12,
+                                 righttitle=U.units,
+                                 righttitlefontsize=12)
+
+# Panel 1: Contourf-plot U data (for shading)
```

Comment:
I think this comment (and the ones for 2 and 3) should be edited to be a bit more specific. Maybe something like:
```suggestion
# Panel 1: Contourf-plot U data with '//' and '..' hatch styles
```
but open to other routes too!

Reply:
Thanks for the suggestions. Makes sense to me!